### ISSUE TITLE
[1.11.2] File minecraft:sounds/tock4.ogg does not exist

### ISSUE THREAD

**Issue by bookerthegeek** [Nuchaz/BiblioCraft#473]
# Heyo # 
Thought you should know, saw this in the console.
```
[08:43:25] [Client thread/WARN]: File minecraft:sounds/tock4.ogg does not exist, cannot add it to event bibliocraft:tock
[08:43:25] [Client thread/WARN]: Invalid sounds.json
java.lang.NullPointerException
	at net.minecraft.client.audio.SoundHandler.func_110549_a(SourceFile:82) [ccs.class:?]
	at net.minecraft.client.resources.SimpleReloadableResourceManager.func_110542_a(SimpleReloadableResourceManager.java:122) [cae.class:?]
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:447) [bes.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:352) [bes.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:124) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_131]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_131]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_131]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_131]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_131]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_131]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_131]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_131]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:236) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:297) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:162) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:53) [NewLaunch.jar:?]
```
---
### Versions ###
- Forge-1.11.2-13.20.0.2366-universal.jar
- BiblioCraft[v2.3.2][MC1.11.2]

**Comment by Nuchaz** [comment]
yeah, I keep seeing this too, but can't figure out what the deal is. tock4.ogg exists and is in the jar, and I can play it in a media player, but it seems to error out for some reason.

**Comment by bookerthegeek** [comment]
Is it correctly identifies on the sound.json?

**Comment by Nuchaz** [comment]
ah, that is what is was. I was missing the bibliocraft: prefix. Only on that one sound though. I'll have a fix in the next release.

**Comment by Nuchaz** [comment]
This is fixed in the latest release for MC versions 1.10, 1.11, and 1.12
